js/arena.js

```diff
@@ -0,0 +1,63 @@
+console.log('its work');
+import gladiator from './gladiator.js';
+export default class arena extends gladiator {
+
+constructor(name ,gladiators = []){
+    super();
+    this.name = name.charAt(0).toUpperCase() + name.slice(1);
```

Comment:
good job with the first character, the rest of the character should be turned into lower case.